### PR TITLE
Updating all pages

### PR DESCRIPTION
**Pull request by ahmadmardeni1**
I've made more than 100 edits, some of them are:
- Grammatical issues.
- Typos.
- Extra/missed commas/dots.

I've spent a lot of time on this and updated all pages, so please let me know if there are any edits required.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
cc @schmidsi

File: pages/en/hosted-service/migrating-subgraph.mdx [hunk 1]
```diff
@@ -139,7 +139,7 @@ While this is not a live feature in the product UI, you can set your maximum bud
...
 
-Remember that it's a dynamic and growing market, but how you interact with it is in your control. There is no maximum or minimum price specified in the protocol or the Gateways. For example, you can look at the price paid by a few of the dapps on the network (on a per-week basis), below. See the last column, which shows query fees in GRT. For example, [Pickle Finance](https://www.pickle.finance/) has 8 requests per second and paid 2.4 GRT for one week.
+Remember that it's a dynamic and growing market, but how you interact with it is in your control. There is no maximum or minimum price specified in the protocol or the Gateways. For example, you can look at the price paid by a few of the dApps on the network (on a per-week basis), below. See the last column, which shows query fees in GRT. For example, [Pickle Finance](https://www.pickle.finance/) has 8 requests per second and paid 2.4 GRT for one week.
```
Comment (azf20) [inline pages/en/hosted-service/migrating-subgraph.mdx:142]: dapp not dApp

File: pages/en/hosted-service/what-is-hosted-service.mdx [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: What is the Hosted Service?
 
 This section will walk you through deploying a subgraph to the Hosted Service, otherwise known as the [Hosted Service.](https://thegraph.com/hosted-service/) As a reminder, the Hosted Service will not be shut down soon. We will gradually sunset the Hosted Service once we reach feature parity with the decentralized network. Your subgraphs deployed on the Hosted Service are still available [here.](https://thegraph.com/hosted-service/)
 
-If you don't have an account on the Hosted Service, you can signup with your Github account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. Graph Node supports a number of Ethereum testnets (Rinkeby, Ropsten, Kovan) in addition to mainnet.
+If you don't have an account on the Hosted Service, you can signup with your Github account. Once you authenticate, you can start creating subgraphs through the UI and deploying them from your terminal. Graph Node supports several Ethereum testnets (Rinkeby, Ropsten, Kovan) in addition to mainnet.
```
Comment (azf20) [inline pages/en/hosted-service/what-is-hosted-service.mdx:7]: I think this sentence understates the number of supported networks at this point, but that is perhaps outside the scope of this PR

File: pages/en/studio/subgraph-studio.mdx [hunk 3]
```diff
@@ -17,18 +17,18 @@ What you can do in the Subgraph Studio:
 
 Here in the Subgraph Studio, you have full control over your subgraphs. Not only can you test your subgraphs before you publish them, but you can also restrict your API keys to specific domains and only allow certain indexers to query from their API keys.
 
-Querying subgraphs generates query fees, used to reward [indexers](/indexing) on the Graph network. If you’re a dapp developer or subgraph developer, the Studio will empower you to build better subgraphs to power your or your community’s queries. The Studio is comprised of 5 main parts:
+Querying subgraphs generates query fees, used to reward [indexers](/indexing) on the Graph network. If you’re a dApp developer or subgraph developer, the Studio will empower you to build better subgraphs to power your or your community’s queries. The Studio is comprised of 5 main parts:
```
Comment (azf20) [inline pages/en/studio/subgraph-studio.mdx:20]: dapp not dApp

File: pages/en/studio/subgraph-studio.mdx [hunk 4]
```diff
@@ -100,7 +100,7 @@ Whenever you deploy a new subgraph version in the Subgraph Studio, the previous
 
 ## Managing your API Keys
 
-Regardless of whether you’re a dapp developer or a subgraph developer, you’ll need to manage your API keys. This is important for you to be able to query subgraphs because API keys make sure the connections between application services are valid and authorized. This includes authenticating the end user and the device using the application.
+Regardless of whether you’re a dApp developer or a subgraph developer, you’ll need to manage your API keys. This is important for you to be able to query subgraphs because API keys make sure the connections between application services are valid and authorized. This includes authenticating the end-user and the device using the application.
```
Comment (azf20) [inline pages/en/studio/subgraph-studio.mdx:103]: dapp not dApp

**Review by azf20 — CHANGES_REQUESTED** [review]
Thanks for the detailed fixes across a lot of files, some eagle-eyed spots there! Some small tweaks (mostly we say dapp not dApp). There might be some conflicts at merge time as this is pretty wide ranging

**Comment by ahmadmardeni1** [comment]
Hey Adam, thank you! @azf20

File: pages/en/studio/subgraph-studio.mdx [hunk 5]
```diff
@@ -17,7 +17,7 @@ What you can do in the Subgraph Studio:
 
 Here in the Subgraph Studio, you have full control over your subgraphs. Not only can you test your subgraphs before you publish them, but you can also restrict your API keys to specific domains and only allow certain indexers to query from their API keys.
 
-Querying subgraphs generates query fees, used to reward [indexers](/indexing) on the Graph network. If you’re a dApp developer or subgraph developer, the Studio will empower you to build better subgraphs to power your or your community’s queries. The Studio is comprised of 5 main parts:
+Querying subgraphs generates query fees, used to reward [indexers](/indexing) on the Graph network. If you’re a dpp developer or subgraph developer, the Studio will empower you to build better subgraphs to power your or your community’s queries. The Studio is comprised of 5 main parts:
```
Comment (azf20) [inline pages/en/studio/subgraph-studio.mdx:20]: small spelling!

**Review by azf20 — CHANGES_REQUESTED** [review]
nice thanks so much! I think there are some conflicts to resolve, could you take a look?

**Comment by ahmadmardeni1** [comment]
Oh sorry, didn't notice the comment. 
Conflicts are resolved. @azf20

**Comment by azf20** [comment]
Thanks @ahmadmardeni1! I still see two conflicts? Sorry if they were more recently introduced

**Comment by ahmadmardeni1** [comment]
Yes, they were recently introduced. No problem, just resolved them! @azf20

**Review by azf20 — APPROVED** [review]

**Pull request merged**